### PR TITLE
Update the documentation to point to the TensorRT backend repo

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ specified in the model's configuration using the 'backend' and
 'platform' settings. Depending on the backend one or the other of
 these properties is optional.
 
-* For TensorRT, 'platform' must be set to *tensorrt_plan*. Currently,
```
Comment (CoderHam) [inline README.md:190]: @Tabrizian probably a good idea to add the python backend in the [backends section](https://github.com/triton-inference-server/backend/blob/main/README.md#backends) as well.

@tanmayv25 same for the onnxruntime backend as well.

**Review by CoderHam — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (deadeyegoodwin) [reply README.md:190]: The backends listed here are for older backends that can still be selected using the "platform" setting. Newer backends (like python and onnxruntime) do not use "plaform" and so do not need to be listed here.

Comment (CoderHam) [reply README.md:190]: Ah yes! Resolving this

**Review by GuanLuo — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Comment by deadeyegoodwin** [comment]
@tanmayv25  I believe there are also some updates needed in the server/ repo docs. Specifically I think compose.md needs updating to mention that tensorrt backend is no longer built into triton.

**Pull request merged**